frontends/web/tests/gapSettings.test.ts

```diff
@@ -0,0 +1,79 @@
+/**
+*  Copyright 2025 Shift Crypto AG
+*
+*  Licensed under the Apache License, Version 2.0 (the "License");
+*  you may not use this file except in compliance with the License.
+*  You may obtain a copy of the License at
+*
+*       http://www.apache.org/licenses/LICENSE-2.0
+*
+*  Unless required by applicable law or agreed to in writing, software
+*  distributed under the License is distributed on an "AS IS" BASIS,
+*  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*  See the License for the specific language governing permissions and
+*  limitations under the License.
+*/
+
+
+
+import { test } from './helpers/fixtures';
+import { ServeWallet } from './helpers/servewallet';
+import { expect } from '@playwright/test';
+import { clickButtonWithText } from './helpers/dom';
+import { deleteAccountsFile } from './helpers/fs';
+
+let servewallet: ServeWallet;
+
+test('Gap limits are correctly saved', async ({ page, host, frontendPort, servewalletPort }) => {
+
+  await test.step('Start servewallet', async () => {
+    servewallet = new ServeWallet(page, servewalletPort, frontendPort, host);
+    await servewallet.start();
+  });
+
+
+  await test.step('Navigate to the app', async () => {
+    await page.goto(`http://${host}:${frontendPort}`);
+    const body = page.locator('body');
+    await expect(body).toContainText('Please connect your BitBox and tap the side to continue.');
+  });
+
+  await test.step('Type into gap limit inputs', async () => {
+    await page.getByRole('link', { name: 'Settings' }).click();
+    await page.getByRole('link', { name: 'Advanced settings' }).click();
+    await page.getByRole('button', { name: 'Custom gap limit settings' }).click();
+
+    const recvInput = page.locator('#gapLimitReceive');
+    await recvInput.clear();
```

Comment:
The Dialog component will focus the first focusable element inside the dialog. Would it make sense to check focus here and also later after the change gap was changed?


https://playwright.dev/docs/api/class-locatorassertions#locator-assertions-to-be-focused